io_scene_godot/converters/mesh.py

```diff
@@ -13,6 +13,99 @@
 
 MAX_BONE_PER_VERTEX = 4
 
+# ------------------------------- Prim -----------------------------------
```

Comment:
Could you move primitive and CSG support into another pull request? Let this pull request just for Grease. Separate PRs can be more effective to get things merged